### PR TITLE
Evalcast cleanup

### PR DESCRIPTION
**Pull request by dajmcdon**
It looks like lots of changes, but not really. Mostly I wanted to clean up the .R files.

1. I moved functions around to files with matching names.
2. I also rebuilt the documentation and the pkgdown site (when I uncovered the vignette fail #449 ).
3. I deleted `plot_measure()`.
4. I stopped exporting `get_target_period()`
5. I exported `plot_canonical()`
6. I cleaned up the multiple/missing final lines on most files.

### CODE REVIEW

File: R-packages/evalcast/R/download_signals.R [hunk 1]
```diff
@@ -64,7 +64,17 @@ download_signals <- function(...,
   for (i in seq_along(msg)) message(msg[i])
   
   out <- base::suppressMessages({covidcast_signals(...)})
+  empty_signals <- purrr::map_lgl(out, ~nrow(.x) == 0 || is.null(.x))
```
Comment (sgsmob) [inline R-packages/evalcast/R/download_signals.R:67]: Should the conditions here be reversed for lazy evaluation purposes?  I assume `nrow(NULL)` is well defined, but it seems better to skip that.

File: R-packages/evalcast/R/empty_.R [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (sgsmob) [inline R-packages/evalcast/R/empty_.R:1]: What is the convention for the trailing underscore in the file name here?  If this is supposed to be a file that isn't user facing, the Python convention is to lead with the underscore, which I find much more readable.

File: R-packages/evalcast/R/get_target_response.R [hunk 3]
```diff
@@ -52,7 +11,7 @@ get_target_response <- function(signals,
     enframe(name = NULL, value = "forecast_date") %>%
     mutate(incidence_period = incidence_period, ahead = ahead) %>%
     pmap_dfr(get_target_period)
-
+
```
Comment (sgsmob) [inline R-packages/evalcast/R/get_target_response.R:14]: Remove trailing spaces here and in 1 other place below.

File: R-packages/evalcast/R/get_target_response.R [hunk 4]
```diff
@@ -103,17 +62,9 @@ get_target_response <- function(signals,
...
-}
-
-empty_actual <- function(){
-  out <- tibble(geo_value = character(0), forecast_date = lubridate::ymd(),
-                actual=double(0), start = lubridate::ymd(), 
-                end = lubridate::ymd()) %>% group_by(.data$geo_value)
-  attr(out, "as_of") <- Sys.Date()
-  out
-}
+}
```
Comment (sgsmob) [inline R-packages/evalcast/R/get_target_response.R:70]: Trailing newline

**Review by sgsmob — APPROVED** [review]

File: R-packages/evalcast/R/get_target_response.R (hunk 3 above)
Comment (dajmcdon) [reply R-packages/evalcast/R/get_target_response.R:14]: I think I got these, but maybe check to be sure.

Comment (sgsmob) [reply R-packages/evalcast/R/get_target_response.R]: LGTM

**Pull request merged**